apps/site/assets/css/_stop-card.scss

```diff
@@ -214,16 +214,17 @@ $radius: 4px;
 
 .departures-container {
   background-color: $white;
+  margin: .5rem;
   @include media-breakpoint-down(sm) {
     box-shadow: 0 0 $space-4 $space-2 $drop-shadow-color;
     height: 100vh;
-    left: 0;
+    left: -.5rem;
```

Comment:
question: Why are these negative margins needed?

Reply:
Without them the overlay wasn't covering the entire screen